lib/posix-process/signal/signal.h

```diff
@@ -0,0 +1,243 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+/* Copyright (c) 2023, Unikraft GmbH and The Unikraft Authors.
+ * Licensed under the BSD-3-Clause License (the "License").
+ * You may not use this file except in compliance with the License.
+ */
+
+#ifndef __UK_SIGNAL_H__
+#define __UK_SIGNAL_H__
```

Comment:
Should this be `__UK_SIGNAL_SIGNAL_H__` for consistency with sigset.h? I don't see where the UK could come from since it is not in a uk dir.

Reply:
Regardless of the directory, the `UK` prefix has to be there to prevent conflicts with guards in libc headers. For the remaining part I chose a more relaxed approach as these are private headers. If we wanted to go by the book this would need to be `__UK_PROCESS_SIGNAL_SIGNAL_H__`  and so on.